apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInAnyOrderOnlyCreatorSamples.kt

```diff
@@ -6,6 +6,28 @@ import kotlin.test.Test
 
 class MapLikeToContainInAnyOrderOnlyCreatorSamples {
 
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a")).toContain.inAnyOrder.only.entry(1 to "a")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.only.entry(1 to "a")
```

Comment:
```suggestion
            expect(mapOf(1 to "a", 2 to "b"))
                .toContain.inAnyOrder.only.entry(1 to "a") // fails because subject has additional entries
```